src/lib.rs

```diff
@@ -99,6 +100,7 @@ impl<T: IoRead + IoWrite> IoReadAndWrite for T {}
 ///
 /// ## Return value
 /// [`TtfbOutcome`] or [`TtfbError`].
+// #[no_panic]
```

Comment:
Nice, I didn't know that! Sounds good

Reply:
Unfortunately I had to remove no-panic because some libraries some of these functions depend on cause panics down the stack